airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -273,7 +247,7 @@ def request_params(
             selected_properties = {
                 key: value
                 for key, value in selected_properties.items()
-                if not (("format" in value and value["format"] == "base64") or ("object" in value["type"] and len(value["type"]) < 3))
+                if not (("format" in value and value["format"] == "base64") or "object" in value["type"])
```

Comment:
what a bloody condition to understand, can we do like this:
```python
if value.get("format") != "base64" and "object" not in value["type"]:
```
seems like the same condition but better to read, could you check it out as well?